<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintWindow</class>
 <widget class="QMainWindow" name="PaintWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgorund">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>851</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/stockes/resources/whiteBackground.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="rectangleBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f9f9f9;
color: black;
border-radius: 20px;
font: 11pt &quot;Montserrat&quot;;</string>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="triangleBtn">
    <property name="geometry">
     <rect>
      <x>573</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f9f9f9;
color: black;
border-radius: 20px;
font: 11pt &quot;Montserrat&quot;;</string>
    </property>
    <property name="text">
     <string>Triangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circleBtn">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f9f9f9;
color: black;
border-radius: 20px;
font: 11pt &quot;Montserrat&quot;;</string>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lineBtn">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f9f9f9;
color: black;
border-radius: 20px;
font: 11pt &quot;Montserrat&quot;;</string>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>18</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/stockes/resources/Logo.png</pixmap>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>71</y>
      <width>1069</width>
      <height>751</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f4f5f5;
border-radius: 20px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>30</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #710101;
text-align: left;
font: 21pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="miniBtn">
    <property name="geometry">
     <rect>
      <x>1075</x>
      <y>30</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #000;
text-align: left;
font: 21pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorBtn">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>150</y>
      <width>26</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #a5aeaf;
text-align: left;
font: 20pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="redoBtn">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #a5aeaf;
text-align: left;
font: 20pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+Y</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tableBtn">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>799</y>
      <width>29</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #a5aeaf;
text-align: left;
font: 28pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>o</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="undoBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>29</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #a5aeaf;
text-align: left;
font: 18pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>u</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+Z</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>370</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: #a5aeaf;
text-align: left;
font: 19pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>f</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>470</y>
      <width>16</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

.QSlider::groove:vertical {
    border: 1px solid #262626;
    width: 3px;
	background: #000000;
}

.QSlider::handle:vertical {
	background:#000000;
    height: 8px;
    margin: 0px -5px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>25</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f9f9f9;
color: black;
border-radius: 15px;
padding: 45px;
font: 9pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchIcon">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>26</y>
      <width>22</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;IROicons&quot;;</string>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ersr_Btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f9f9f9;
color: black;
border-radius: 20px;
font: 11pt &quot;Montserrat&quot;;</string>
    </property>
    <property name="text">
     <string>Eraser</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="DataTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>150</y>
      <width>329</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SortDSBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>120</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Dscending</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SortASBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>120</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Ascending</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="InfoTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>620</y>
      <width>329</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>PaintWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1112</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>1161</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>miniBtn</sender>
   <signal>clicked()</signal>
   <receiver>PaintWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1079</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>1151</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
